integration_tests/__init__.py

```diff
@@ -212,6 +216,108 @@ def set_package_version(
         return version
 
 
+class BzrSourceBaseTestCase(TestCase):
+
+    def setUp(self):
+        super().setUp()
+        if shutil.which('bzr') is None:
+            self.skipTest('bzr is not installed')
+
+    def init_and_config_bzr(self):
+        subprocess.check_call(
+            ['bzr', 'init', '.'],
+            stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
+        subprocess.check_call(
+            ['bzr', 'whoami', '--branch', '"Example Dev <dev@example.com>"'])
+
+    def commit(self, message, unchanged=False):
+        command = ['bzr', 'commit', '-m', message]
+        if unchanged:
+            command.append('--unchanged')
+        subprocess.check_call(
+            command, stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
+
+    def get_revno(self, path=None):
+        command = ['bzr', 'revno', '-r', '-1']
+        if path:
+            command.append(path)
+        return subprocess.check_output(
+            command, universal_newlines=True).strip()
+
+
+class GitSourceBaseTestCase(TestCase):
+
+    def setUp(self):
+        super().setUp()
+        if shutil.which('git') is None:
+            self.skipTest('git is not installed')
+
+    def init_and_config_git(self):
```

Comment:
And I think we can just call it init, remove the config part. But I'd prefer to leave it in a single function, at least until we find a test that requires init but not the part setting the email.